src/compat/compat.h

```diff
@@ -34,9 +34,8 @@
  * To compile with PG16, use -DEXPERIMENTAL=ON with cmake.
  */
 #define is_supported_pg_version(version)                                                           \
-	(is_supported_pg_version_12(version) || is_supported_pg_version_13(version) ||                 \
-	 is_supported_pg_version_14(version) || is_supported_pg_version_15(version) ||                 \
-	 is_supported_pg_version_16(version))
+	(is_supported_pg_version_13(version) || is_supported_pg_version_14(version) ||                 \
+	 is_supported_pg_version_15(version) || is_supported_pg_version_16(version))
 
 #define PG12 is_supported_pg_version_12(PG_VERSION_NUM)
```

Comment:
do we want to keep this? it seems there are a few occurances of `PG12` in this file - shouldn't that be removed as well?

Reply:
Yes they have to be removed but I'm planning to do that in a separate PR. I have mentioned that in the commit message as well.

Reply:
I've missed the comment in the commit message - the PR has 2 commits right now....

but sure...ok to address it later as well!